torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -1313,13 +1313,92 @@ std::vector<For*> LoopNest::distributeLoopOverInnerLoops(For* loop) {
   return distributeLoop(loop, loopsSet);
 }
 
+// TODO: Move these functions to a separate util file.
```

Comment:
Nit, but IMO let's save TODO comments for stuff that's really notable.

Reply:
Removed.